### ISSUE TITLE
Feature request: Creating sparse or partial indizes in mongodb

### ISSUE THREAD

**Issue by svkurowski** [db-migrate/node-db-migrate#522]
Hello, thanks for producing this awesome module.

It would be great to be able to create [sparse](https://docs.mongodb.com/manual/core/index-sparse/) and/or [partial](https://docs.mongodb.com/manual/core/index-partial/) indizes in Mongodb.

A workaround would be to implement #358.

**Comment by svkurowski** [comment]
Please re-open this, it makes no sense to close feature requests as stale due to lack of activity. I'm still interested in the features described above!

**Comment by wzrdtales** [comment]
@svkurowski It totally makes sense, if you don't want this to be closed, just show activity and the bot wont close the issue. This is the whole sense after it, to weed out, but I'm very open to reopen any issue that was closed due to inactivity.

Also we filter for some tags, but this issue just got no tags yet, which I will change now.

See 
https://github.com/db-migrate/node-db-migrate/blob/5450eb4dc2778d2f379b115a7158233d9b55dbf1/.github/stale.yml#L7

**Comment by wzrdtales** [comment]
Also to note there is currently an open PR that would allow access to the native mongodb api.

**Comment by svkurowski** [comment]
@wzrdtales Thank you for re-opening this. I guess we just disagree on closing issues like this one in particular but this is your project and I'm using your work for free so I'm not going to try to tell you how to do things, I'm just interested in the feature I requested here.

I don't see any open PR but look forward to seeing that feature implemented.

**Comment by wzrdtales** [comment]
Yes b/c this is the core project and not the driver which you are requesting a feature for. And no not a disagreement, features should not be closed, but your issue was simply not yet tagged, so it was closed and it was good that it was closed, b/c it raised the attention again.

The repository for the driver is here:

https://github.com/db-migrate/mongodb
